gapic-common/lib/gapic/common/retry_policy.rb

```diff
@@ -0,0 +1,196 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "gapic/common/error_codes"
+
+module Gapic
+  module Common
+    # Gapic Common retry policy base class.
+    class RetryPolicy
+      DEFAULT_INITIAL_DELAY = 1
+      DEFAULT_MAX_DELAY = 15
+      DEFAULT_MULTIPLIER = 1.3
+      DEFAULT_RETRY_CODES = [].freeze
+      DEFAULT_TIMEOUT = 3600 # One hour
```

Comment:
Each of the above constants should have a descriptive comment similar to a method's documentation, because they are public.